components/jupyter-web-app/backend/kubeflow_jupyter/common/utils.py

```diff
@@ -263,6 +263,19 @@ def process_resource(rsrc, rsrc_events):
     # VAR: change this function according to the main resource
     cntr = rsrc["spec"]["template"]["spec"]["containers"][0]
     status, reason = process_status(rsrc, rsrc_events)
+    
+    # GPUs may not have been assigned to a pod 
```

Comment:
nit: @lalithvaka could you add this logic into a distinct function on its own?

Reply:
@kimwnasptd submitted the change requested.